groovy/postInit/mod/Supercritical.groovy

```diff
@@ -54,18 +59,290 @@ ASSEMBLER.recipeBuilder()
     .inputs(metaitem('hull.ev'))
     .inputs(ore('circuitEv') * 8)
     .inputs(metaitem('plate.power_integrated_circuit') * 4)
-    .inputs(ore('pipeSmallFluidAluminium') * 16)
+    .inputs(ore('pipeSmallFluidStainlessSteel') * 16)
     .inputs(metaitem('electric.pump.ev') * 4)
-    .circuitMeta(22)
-    .outputs(item('gregtech:machine', 14403))
+    .circuitMeta(30)
+    .outputs(metaitem('supercritical:gas_centrifuge'))
     .duration(240)
-    .EUt(Globals.voltAmps[1])
+    .EUt(Globals.voltAmps[4])
     .buildAndRegister()
 
-RecyclingHelper.handleRecycling(item('gregtech:machine', 14403),
+RecyclingHelper.handleRecycling(metaitem('supercritical:gas_centrifuge'),
     [metaitem('hull.ev'),
     ore('circuitEv') * 8,
     metaitem('plate.power_integrated_circuit') * 4,
-    ore('pipeSmallFluidAluminium') * 16,
+    ore('pipeSmallFluidStainlessSteel') * 16,
     metaitem('electric.pump.ev') * 4]
-)
\ No newline at end of file
+)
+
+// Spent Fuel Pool Controller
+ASSEMBLER.recipeBuilder()
+    .inputs(metaitem('hull.mv'))
+    .inputs(ore('circuitMv') * 4)
+    .inputs(ore('pipeSmallFluidAluminium') * 16)
+    .inputs(metaitem('electric.pump.mv') * 16)
+    .circuitMeta(30)
+    .outputs(metaitem('supercritical:spent_fuel_pool'))
+    .duration(120)
+    .EUt(Globals.voltAmps[3])
+    .buildAndRegister()
+
+RecyclingHelper.handleRecycling(metaitem('supercritical:spent_fuel_pool'),
+    [metaitem('hull.mv'),
+    ore('circuitMv') * 4,
+    ore('pipeSmallFluidAluminium') * 16,
+    metaitem('electric.pump.mv') * 16]
+)
+
+// Panelling
+ASSEMBLER.recipeBuilder()
+    .inputs(ore('plateSteel') * 4)
+    .circuitMeta(17)
+    .outputs(item('supercritical:panelling', 7))
+    .EUt(16)
+    .duration(120)
+    .buildAndRegister()
+
+for (int i = 0; i < CHEMICAL_DYES.length; i++) {
+    CHEMICAL_BATH.recipeBuilder()
+        .inputs(item('supercritical:panelling', 7))
+        .fluidInputs(CHEMICAL_DYES[i].getFluid(9))
+        .outputs(item('supercritical:panelling', i))
+        .EUt(2).duration(10)
+        .buildAndRegister()
+
+    RecyclingHelper.handleRecycling(item('supercritical:panelling', i),
+        [ore('plateSteel') * 4]
+    )
+}
+
+// Fission Reactor Controller
+ASSEMBLER.recipeBuilder()
```

Comment:
this recipe is kinda goofy

Reply:
yeah it's just some sensors + red alloy wires for signal wiring